.gitignore

```diff
@@ -9,14 +9,16 @@ config/config.json
 /client/build
 
 # working dir
-/wp-calypso
+/server/repository
 
 # misc
 .DS_Store
+.env
```

Comment:
`docker-compose` picks this file automatically and sets env variables visible in containers.